content/platforms/kubernetes/quick-start.md

```diff
@@ -0,0 +1,164 @@
+---
+Title: Quick Start for Kubernetes
+description: The Redis Enterprise Operator is a deployment that must be
+  installed on your cluster to facilitate Redis Enterprise cluster deployments. Installing the operator is
+  a simple process.
+weight: 10
+alwaysopen: false
+categories: ["Platforms"]
+aliases:
+---
+
+The Redis Enterprise Operator is a software component that must be
+installed first to facilitate cluster deployments. Some environments provide
+ways to install operators through managed catalogs. For example, the
+[OpenShift OperatorHub](https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html)
+provides an administration interface by which you can find and install the Redis Enterprise
+Operator.
+
+If you are using OpenShift, see [Getting Started with OpenShift and Kubernetes]({{< relref "/platforms/kubernetes/openshift/_index.md" >}}) for
+installation information
+
+If you are using PKS, see [Getting Started with PKS (Pivotal Container Service)]({{< relref "/platforms/kubernetes/pks/_index.md" >}}) for
+installation information.
+
+Otherwise, the operator can be installed on a supported Kubernetes platform
+directly by the following procedure outlined here.
+
+## Overview of the operator
+
+The operator acts on [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRD) that describe Redis Enterprise clusters and databases. The operator itself is a deployment, backed by pods, that runs within your deployment namespace. These operator pods must run with sufficient privileges to create the Redis Enterprise cluster resources within that namespace.
+
+When installed, the following are created:
+
+ * a service account under which the operator will run,
+ * a set of Roles that define the privileges necessary for the operator to perform its tasks,
+ * a set of Role Bindings to the service account to authorize the service account to have the correct roles (see above),
+ * the CRD for a Redis Enterprise Cluster,
+ * the CRD for a Redis Enterprise Database,
+ * the operator (a deployment).
+
+ The operator currently runs within single namespace and is scoped to operate only on cluster CRD's in that namespace. This allows a multi-tenant deployment of the operator supporting different versions in different namespaces.
```

Comment:
```suggestion
 The operator currently runs within a single namespace and is scoped to operate only on Redis Enterprise Cluster CRD's in that namespace. This allows the deployment of the multiple instances of the operator on the same Kubernetes cluster, supporting multiple Redis Enterprise Clusters in multiple namespaces.
```
Multi-tenancy is ambiguous as it may refer to K8s cluster or Redis Enterprise Cluster. We need to find a way to promote the pattern of multiple DBs on the cluster while separately discussing the benefits or multiple Enterprise clusters... regarding versions - to a degree - in that the CRD at the K8s cluster level may limit which version you can deploy - see if you can work that into here or maybe at this level we keep it simple and just mention that the operator is namespace scoped.